README.md

```diff
@@ -99,6 +99,13 @@ You can deploy this project using: **Not available**.
 - Twitter: [@jeanbuhendwa](https://twitter.com/jeanjacqueI)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/johnbuhendwa/)
 
+👤 **Damilare Adepoju**
+
+- GitHub: [@githubhandle](https://github.com/adamilare)
+- Twitter: [@twitterhandle](https://twitter.com/mailtodare)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/adamilare)
```

Comment:
- [ ] Kindly fix the broken link to your `LinkedIn` account, as broken links are not allowed
![image](https://github.com/jeanbuhendwa/vet-clini-db/assets/81932478/fadb7aa5-afb0-4e49-bdc4-8ca80e721902)